tensorflow/lite/tools/optimize/operator_property.cc

```diff
@@ -92,6 +92,12 @@ OperatorProperty GetOperatorProperty(const ModelT* model, int subgraph_index,
       property.restrict_same_input_output_scale = true;
       property.version = 2;
       break;
+    case BuiltinOperator_SPLIT_V:
+      property.inputs = {{0, {}}};
+      property.outputs = {{0, {}}};
+      property.restrict_same_input_output_scale = true;
```

Comment:
Should property.arbitrary_outputs be set here as well, similar to Split op above?

Reply:
thanks for spotting this! corrected.

Reply:
Thanks!